WindowsServerDocs/networking/technologies/pktmon/pktmon-syntax.md

```diff
@@ -74,22 +75,28 @@ C:\Test> pktmon filter add -i 10.0.0.10 -t tcp syn
 
 For more information, see [pktmon filter syntax](/windows-server/administration/windows-commands/pktmon-filter).
 
-## Packet capture and logging
+## Packets and general events capture
+PacketMon can capture packets through the [-c] parameter with the start command. This will enable packet capture and logging as well as packet counters. To enable packet counters only without logging the packet, add the [-o] parameter to the start command. For more information about packet counters, please check the Packet Counters section below.
 
-Packet Monitor can capture networking traffic with or without packet logging. For more information about capturing traffic without packet logging, check the Packet Counters section below. To capture and log packets, add the **[--etw]** parameter to the start command.
+You can select components to monitor through the [--comp] parameter. It can be NICs only, or a list of component ids, and it defaults to all components. You can also filter by packet propagation status (dropped or flowing packets) through the [--type] parameter.
```

Comment:
minor detail (capitalization):
```suggestion
You can select components to monitor through the [--comp] parameter. It can be NICs only, or a list of component IDs, and it defaults to all components. You can also filter by packet propagation status (dropped or flowing packets) through the [--type] parameter.
```